syncstorage/tests/test_sql.py

```diff
@@ -197,3 +198,21 @@ def count_items():
         self.assertEquals(count_items(), 5)
         self.assertEquals(len(self.storage.get_items(_USER, "col")["items"]),
                           5)
+
+    def test_migrating(self):
+        migrating_id = str(uuid.uuid4())
+        with self.storage.dbconnector.connect() as connect:
+            sql = """
+                INSERT INTO migration
+                    (fxa_uid, started)
+                VALUES
+                    ('{}', {})
+                /* queryName=migrate */""".format(
```

Comment:
nit: I know it's just a test, but you could use sql parameters here rather than string formatting:

```
connect.execute(sql, params={"fxa_uid": migrating_id, "started": int(time.time())})
```